### PR TITLE
Purva's card

### PR DESCRIPTION
**Pull request by purva-g**
Insertion of card to index.html file through altering links.

### CODE REVIEW

**Review by alsonick — CHANGES_REQUESTED** [review]
Your commit is trying to change lines `3100`-`3111` in the `index.html` file which should not be the case. You need to start your card template on line `117` as stated in the [instructions](https://github.com/Syknapse/Contribute-To-This-Project#readme). Also please include the resources section containing 3 links to resources that are useful for web development as well. If you need any help then please comment below, once you've made the necessary changes I can merge your pr. Thanks! 😄

**Comment by purva-g** [comment]
Changes have been made on line `117`. **Resources section added** along with 3 useful links for a beginner in WebDev. Thank You.

File: index.html [hunk 1]
```diff
@@ -114,7 +114,32 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+		   <!-- Purva Gharat card START -->
```
Comment (alsonick) [inline index.html:117]: Just fix this minor indentation then you should be good to go!

**Review by alsonick — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (purva-g) [reply index.html]: Indentation removed ! Thank You.

Comment (alsonick) [reply index.html]: Thank you, merged your pr!

**Pull request merged**